app/collins/models/lshw/ServerBase.scala

```diff
@@ -22,9 +26,9 @@ object ServerBase {
 }
 
 case class ServerBase(
-    description: String = "", product: String = "", vendor: String = "", serial: Option[String] = None) extends LshwAsset {
+    motherboard: String = "", firmware: String = "", description: String = "", product: String = "", vendor: String = "", serial: Option[String] = None) extends LshwAsset {
```

Comment:
why not make these options instead of strings with empty defaults?

Reply:
Sounds good to me, changed in e64829b54711877d6342e15b8e9019a4b0440eb8.